spec/validator_spec.rb

```diff
@@ -188,6 +188,21 @@
     check[:ttl] = 1
     @validator.validate_check(check)
     expect(@validator.reset).to eq(0)
+    check[:aggregates] = "string"
+    @validator.validate_check(check)
+    expect(@validator.reset).to eq(1)
```

Comment:
@portertech any ideas why this test is not passing? I tried adding an explicit case for `is_a_string?` but no dice.